src/components/Icon/IconCommon.tsx

```diff
@@ -206,6 +207,7 @@ export const ICONS_MAP = {
   "folder-open": FolderOpen,
   gear: Gear,
   gift: Gift,
+  globe: Globe,
```

Comment:
you'll need to add it here too https://github.com/ClickHouse/click-ui/blob/739d7f86e60adfa7793f64440d6a214b81a464b7/src/components/Icon/types.ts#L10